sdk/typescript/src/cryptography/keypair.ts

```diff
@@ -22,4 +31,19 @@ export interface Keypair {
    * Get the key scheme of the keypair: Secp256k1 or ED25519
    */
   getKeyScheme(): SignatureScheme;
+
+  export(): ExportedKeypair;
+}
+
+export function fromExportedKeypair(keyPair: ExportedKeypair): Keypair {
+  const secretKey = fromB64(keyPair.privateKey);
+  const publicKey = fromB64(keyPair.publicKey);
+  switch (keyPair.schema) {
+    case 'ED25519':
+      return new Ed25519Keypair({ secretKey, publicKey });
```

Comment:
@joyqvq @kchalkias do you think we could use only the private key and create the keypair from the `fromSecretKey` function? Which approach is better?

Reply:
we should ideally create the pubkey from privkey instead of trusting the pubkey imported

Reply:
Ok will update it

Reply:
Yes, that's the best approach.